home/ivp-planning/advanced-tools/dxf-tool.md

```diff
@@ -0,0 +1,80 @@
+{% endhint %}
+
+{% hint style="info" %}
+For better understanding and training we created a [<img src="../../../.gitbook/assets/YouTube_icon.png" alt="" data-size="line"> YouTube Tutorial](https://www.youtube.com/watch?v=Maxivv824Dk&list=PLlzoGkRUR67houzn5F5ejD3R-kQrDcps5&index=18), where you can find a detailed explanation how to import and export of DXF files.
+{% endhint %}  // Video TBD
+
+## Importing a DXF File:
+
+To import a DXF click **File -> DXF -> Import DXF**. 
+
+![](../../../.gitbook/assets/DXF\_Import\_open.jpg)
+
+A new window will open that allows to navigate to the folder containing the DXF file to be imported. to open the file, simply double click it **or** select it and click Open.
+
+![](../../../.gitbook/assets/DXF\_Import\_file.jpg)
+
+Before iVP Planning imports a DXF file, a prompt will ask to set some parameters that specify how the DXF file will be imported. 
+
+![](../../../.gitbook/assets/DXF\_Import\_settings.jpg)
+
+* _**File path:**_ this line represents the location of the to be imported file, change it by clicking the three dots
+* _**As references:**_ this toggle defines whether or not you reference the dxf file. If checked, you will need to keep the dxf file accessible. (!changing the name or location of that file will break the reference!) 
+* _**Line width:**_ this value represents the width each single line will be drawn (in mm)
+* _**Scale:**_ this value represents the scale **of the DXF to import**, e.g. 1:1 or 16:1
+* _**Unit:**_ this value represents the measurement unit **of the DXF to import**, e.g. mm, cm, inch...
+
+![](../../../.gitbook/assets/DXF\_Import\_comparison.jpg)
+
+{% hint style="info" %}
+Note that the **scale** and the **measurement unit** for imported files refer to the scale/unit the DXF file was planned in and not the scale you want to use in your iVP Planning project. If changed in comparison to the origin; the size of the import might be incorrect.
+{% endhint %}
+
+## Exporting a DXF File:
+
+To export objects from iVP Planning, first select the ojects you want to export.
+
+![](../../../.gitbook/assets/DXF\_Export\_selection.jpg)
+
+Then click **File -> DXF -> Export Selection**.
+
+![](../../../.gitbook/assets/DXF\_Export\_open.jpg)
+
+A new window will open that allows to navigate to the folder the DXF file is to be exported, rename it and save it.
+
+![](../../../.gitbook/assets/DXF\_Export\_file.jpg)
+
+Before iVP Planning exports a DXF file, a prompt will ask to set some parameters that specify how the DXF file will be exported. 
+
+![](../../../.gitbook/assets/DXF\_Export\_settings.jpg)
+
+
+* _**File path:**_ this line represents the location of the to be exported file, change it by clicking the three dots
+* _**Unit:**_ this value represents the measurement unit **of the DXF to export**, e.g. mm, cm, inch...
+* _**Side:**_ this represents the direction the file will be exported to (Overhead View, Side View right, Front View)
```

Comment:
should probably be "from" instead of "to". Maybe "this represents the viewing direction the file will be created from" would be better?

Reply:
agreed, changed ccordingly.